FAQ.md

```diff
@@ -189,16 +187,15 @@ Due to a bug in Android this is not possible.
 See [here](https://github.com/M66B/NetGuard/issues/42) for more information.
 
 <a name="FAQ24"></a>
-**(24) Can you remove the notification from the status bar?**
+**(24) Can you remove the notification from the notification screen?**
```

Comment:
No not that, the thing is as long as you don't actually have an icon htere but only a notification somewhere else ( aka notification screen ), I felt this needed better wording.